lib/Cake/TestSuite/ControllerTestCase.php

```diff
@@ -109,6 +109,7 @@ public function afterRender($viewFile) {
  * ControllerTestCase class
  *
  * @package       Cake.TestSuite
+ * @method		  mixed testAction() testAction($url, $options = array())  Lets you do functional tests of a controller action.
```

Comment:
By "both lines" do you mean line 111 as well as the one that I added? I didn't touch that one. But it does look like my IDE set-up has done something funny to the spacing :)